supervisor/dbus/udisks2/filesystem.py

```diff
@@ -0,0 +1,46 @@
+"""Interface to UDisks2 Filesystem over D-Bus."""
+from typing import Any
+
+from ..const import DBUS_ATTR_MOUNTPOINTS, DBUS_ATTR_SIZE, DBUS_IFACE_FILESYSTEM
+from ..interface import dbus_property
+from ..utils import dbus_connected
+from .block import UDisks2Block
+
+
+class UDisks2Filesystem(UDisks2Block):
+    """Handle D-Bus interface for UDisks2 filesystem device object.
+
+    http://storaged.org/doc/udisks2-api/latest/gdbus-org.freedesktop.UDisks2.Filesystem.html
+    """
+
+    @property
+    @dbus_property
+    def mountpoints(self) -> str:
+        """Return mountpoints."""
+        return [
+            bytes(mountpoint).decode()
+            for mountpoint in self.properties[DBUS_ATTR_MOUNTPOINTS]
+        ]
+
+    @property
+    @dbus_property
+    def size(self) -> int:
+        """Return size."""
+        return self.properties[DBUS_ATTR_SIZE]
+
+    @dbus_connected
+    async def mount(self, options: dict[str, Any]):
+        """Mount filesystem."""
+        await self.dbus.Filesystem.Mount(("a{sv}", options))
```

Comment:
```suggestion
        await self.dbus.Filesystem.call_mount(options)
```
Similar to comments above, to use a dictionary as an input each value must be a `Variant` with the right signature.

Reply:
Suggested changes are accounted for. Leaving open because we need to adjust the dictionaries to have a `Variant` as the value for each key.